app-to-app-js/client_bob.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="./node_modules/@vonage/client-sdk/dist/vonageClientSDK.min.js"></script>
+</head>
+
+<body>
+  <h1>Inbound app call</h1>
+  <p id="notification">Lines are open for calls...</p>
+  <br />
+  <button id="button">Answer</button>
+  <script>
+
+    const token = 'ALICE_JWT';
+    const client = new vonageClientSDK.VonageClient();
```

Comment:
```suggestion
    const client = new vonageClientSDK.VoiceClient();
```
from https://www.npmjs.com/package/@vonage/client-sdk